sw/device/silicon_creator/lib/cert/tpm_ek.hjson

```diff
@@ -0,0 +1,99 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+    name: "tpm_ek",
+
+    variables: {
+        // Creator public key: this is an EC point on
+        // the prime256v1 curve so it has two components
+        // (x and y) which are 32-bytes integers.
+        creator_pub_key_ec_x: {
+            type: "integer",
+            size: 32,
+        },
+        creator_pub_key_ec_y: {
+            type: "integer",
+            size: 32,
+        },
+        // Creator public key ID: this is a 20-byte hash
+        // derived from the creator public key.
+        creator_pub_key_id: {
+            type: "byte-array",
+            size: 20,
+        },
+        // Authority key ID: this is the identifier of
+        // the key that will be used to sign this certificate
+        // offline.
+        auth_key_key_id: {
+            type: "byte-array",
+            size: 20,
+        },
+        // Certificate signature: the result of signing with ECDSA
+        // are two integers named "r" and "s"
+        cert_signature_r: {
+            type: "integer",
+            size: 32,
+        },
+        cert_signature_s: {
+            type: "integer",
+            size: 32,
+        },
+        sal_manufacturer: {
+            type: "string",
+            size: 100,
+        },
+        sal_model: {
+            type: "string",
+            size: 100,
+        },
+        sal_version: {
+            type: "string",
+            size: 100,
+        },
+    },
+
+    certificate: {
+        serial_number: { var: "creator_pub_key_id", convert: "big-endian" },
+        issuer: {
+            country: "USA",
+            state: "CA",
+            organization: "Google",
+            organizational_unit: "Engineering",
+            common_name: "ChromeOS OT ICA",
+        },
+        not_before: "20230101000000Z",
+        not_after: "20500101000000Z",
+        subject: {
+            country: "USA",
+            state: "CA",
+            organization: "Google",
+            organizational_unit: "Engineering",
+            common_name: "ChromeOS OT TPM EK",
+        },
+        subject_public_key_info: {
+            algorithm: "ec-public-key",
+            curve: "prime256v1",
+            public_key: {
+                x: { var: "creator_pub_key_ec_x" },
+                y: { var: "creator_pub_key_ec_y" },
+            },
+        },
+        authority_key_identifier: { var: "auth_key_key_id" },
+        subject_key_identifier: { var: "creator_pub_key_id" },
+        basic_constraints: { ca: false },
+        subject_alt_name: {
+            vendor: { var: "sal_manufacturer" },
+            model: { var: "sal_model" },
+            version: { var: "sal_version" },
+        },
+        extensions: [],
```

Comment:
@pamaury : just a thought for better clarity / consistency with the [X.509 spec](https://datatracker.ietf.org/doc/html/rfc5280#section-4.2), perhaps in a follow up PR we should name the `extensions` field here `private_extensions` since the spec states "_The X.509 v3 certificate format also allows communities to define private extensions to carry information unique to those communities._"  Thoughts?